coldfront/core/user/forms.py

```diff
@@ -273,6 +274,24 @@ def __init__(self, *args, **kwargs):
         self.fields['email_address'].queryset = EmailAddress.objects.filter(
             user=user, is_verified=True, is_primary=False)
 
+class DepartmentSelectionForm(forms.Form):
```

Comment:
I'd move this form (and possibly the other one) to a new `forms` module in `core.department`.

Reply:
moved this one 👍